homeassistant/const.py

```diff
@@ -271,6 +271,7 @@
 EVENT_HOMEASSISTANT_FINAL_WRITE: EventType[NoEventData] = EventType(
     "homeassistant_final_write"
 )
+EVENT_LABS_UPDATED: Final = "labs_updated"
```

Comment:
There's a non-merged proposal to add a labs-helper https://github.com/home-assistant/core/pull/157315 where importing `EVENT_LABS_UPDATED` on module level does not cause circular imports